src/components/NFTCard/utils.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { BodyShape, NFT, NFTCategory, Rarity } from '@dcl/schemas'
+import { Profile } from '../Profile/Profile'
+import { Badge, NFTCardI18N } from './NFTCard.types'
+
+const DEFAULT_BADGE_COLOR = '#37333d'
+
+function _getEmoteBadges(emote: NFT['data']['emote'], i18n: NFTCardI18N) {
+  return [
+    { color: Rarity.getColor(emote.rarity), label: emote.rarity },
+    {
+      color: DEFAULT_BADGE_COLOR,
+      label: emote.loop ? i18n.playMode.loop : i18n.playMode.once,
+      icon: emote.loop
+        ? 'dui-nft-card__badge-icon--loop'
+        : 'dui-nft-card__badge-icon--once',
+      hideLabel: true
+    },
+    ...(emote.hasSound
+      ? [
+          {
+            color: DEFAULT_BADGE_COLOR,
+            label: i18n.withSound,
+            icon: 'dui-nft-card__badge-icon--sound',
+            hideLabel: true
+          }
+        ]
+      : [])
+  ]
+}
+
+function _getWearableBadges(
+  wearable: NFT['data']['wearable'],
+  i18n: NFTCardI18N
+) {
+  const bodyShape =
+    wearable.bodyShapes.length === 2
+      ? 'unisex'
+      : wearable.bodyShapes[0] === BodyShape.MALE
+      ? 'male'
+      : 'female'
+  return [
+    { color: Rarity.getColor(wearable.rarity), label: wearable.rarity },
+    {
+      color: DEFAULT_BADGE_COLOR,
+      label: i18n.category[wearable.category],
+      icon: `dui-nft-card__badge-icon--${wearable.category}`,
+      hideLabel: true
+    },
+    {
+      color: DEFAULT_BADGE_COLOR,
+      label: i18n.bodyShape[bodyShape],
+      icon: `dui-nft-card__badge-icon--${bodyShape}`,
+      hideLabel: true
+    },
+    ...(wearable.isSmart
+      ? [
+          {
+            color: DEFAULT_BADGE_COLOR,
+            label: i18n.smart,
+            icon: 'dui-nft-card__badge-icon--smart',
```

Comment:
Should we use `__` as other css files do?
```suggestion
            icon: 'dui-nft-card__badge-icon__smart',
```
The same applies to all other classes in this file.

Reply:
same as before. It is used as a modifier